_sass/uno.scss

```diff
@@ -1148,9 +1149,11 @@ hr {
 	color: #c7c7c7; }
 
 .post-meta__date {
-	margin-right: .5em; }
+	margin-right: 0;
+	text-transform: lowercase; }
 
 .post-meta__tags {
+	padding: 0 !important;
```

Comment:
Can you make this change in a different place without `!important`?

Reply:
Removed